docs/source/llms/tracing/index.rst

```diff
@@ -97,6 +97,11 @@ Tracing provides a way to record the inputs, outputs, and metadata associated wi
                     <img src="../../_static/images/logos/instructor-logo.svg" alt="Instructor Logo"/>
                 </div>
             </a>
+            <a href="#automatic-tracing">
+                <div class="logo-card">
+                    <img src="../../_static/images/logos/txtai-logo.svg" alt="Txtai Logo"/>
```

Comment:
Let's fix the logo:

<img width="378" alt="image" src="https://github.com/user-attachments/assets/10607521-b759-450f-8d65-d0da9d4ffbe5" />